apps/api/src/server/app/case-officer/appeal-formatter.js

```diff
@@ -1,53 +1,50 @@
 import { addWeeksToDate } from '../utils/add-weeks-to-date.js';
+import formatAddress from '../utils/address-formatter.js';
+import { arrayOfStatusesContainsString } from '../utils/array-of-statuses-contains-string.js';
+import formatDate from '../utils/date-formatter.js';
 import { getAppealStatusCreatedAt } from '../utils/get-appeal-status-created-at.js';
-import { weeksReceivingDocuments } from '../state-machine/full-planning-appeal.machine.js';
+import formatReviewQuestionnaire from '../utils/review-questionnaire-formatter.js';
 
-const add2Weeks = function (date) {
+
+const add2Weeks = ( date) => {
 	const newDate = new Date(date.valueOf());
+
 	newDate.setDate(newDate.getDate() + 14);
 	return newDate;
 };
 
 /**
- * @param {Array} status appeal status array
+ * @param {object} appealStatuses
  * @returns {string} reformatted appeal status
  */
 function mapAppealStatus(appealStatuses) {
-	if (arrayOfStatusesContainsString(appealStatuses, [appealStates.awaiting_lpa_questionnaire])) {
-		return 'awaiting';
-	} else if (arrayOfStatusesContainsString(appealStatuses, [appealStates.overdue_lpa_questionnaire])) {
-		return 'overdue';
-	} else if (arrayOfStatusesContainsString(appealStatuses, [appealStates.received_lpa_questionnaire])) {
-		return 'received';
-	} else if (arrayOfStatusesContainsString(appealStatuses, [appealStates.incomplete_lpa_questionnaire])) {
-		return 'incomplete_lpa_questionnaire';
-	} else {
-		return '';
-	}
+	if (arrayOfStatusesContainsString(appealStatuses, [appealStates.awaiting_lpa_questionnaire])) return 'awaiting';
+	if (arrayOfStatusesContainsString(appealStatuses, [appealStates.overdue_lpa_questionnaire])) return 'overdue';
+	if (arrayOfStatusesContainsString(appealStatuses, [appealStates.received_lpa_questionnaire])) return 'received';
+	if (arrayOfStatusesContainsString(appealStatuses, [appealStates.incomplete_lpa_questionnaire])) return 'incomplete_lpa_questionnaire';
+	return '';
 }
 
 /**
- * @param {Array} status appeal status array
+ * @param {object} appealStatusesParallel
  * @returns {string} reformatted appeal status
  */
 function mapAppealParallelStatuses(appealStatusesParallel) {
-	if (arrayOfStatusesContainsString(appealStatusesParallel, [appealStates.available_for_statements])) {
-		return 'available_for_statements';
-	} else if (arrayOfStatusesContainsString(appealStatusesParallel, [appealStates.available_for_final_comments])) {
-		return 'available_for_final_comments';
-	} else {
-		return '';
-	}
+	if (arrayOfStatusesContainsString(appealStatusesParallel,[appealStates.available_for_statements])) return 'available_for_statements';
+	if (arrayOfStatusesContainsString(appealStatusesParallel, [appealStates.available_for_final_comments])) return 'available_for_final_comments';
+	return '';
 }
 
 const appealFormatter = {
-	formatAppealForAllAppeals: function (appeal) {
+	/**
+	 * @param {{ appealStatus: object; id: any; reference: any; address: object; startedAt: any; }} appeal
```

Comment:
Can we define a custom type for appealStatus as it seems we are repeating this definition a few times?

Reply:
this is now solved too